hack/tools/parse-feature-gates/parse-feature-gates.go

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Would it be better to also move this file into integration? Seems a bit weird for an integration test script to reference hack